### DISCUSSION TITLE
TON Metaspace

### DISCUSSION THREAD

**Discussion by TON-Metaspace** [tonmetaspace/.github#8] [Announcements]
![image](https://user-images.githubusercontent.com/124897831/217838956-0af36829-7ad9-4fc6-af97-54b3a393fca1.png)

🧑‍🚀Our team has prepared several articles describing TON ecosystem main themes:

✅️ [TON Blockchain presentation](https://telegra.ph/The-Open-Network-presentation-12-23)
✅️ [How to register Tonkeepe](https://telegra.ph/How-to-register-Tonkeeper-11-30)
✅️ [How to buy TON in Cryptobot](https://telegra.ph/How-to-buy-TON-in-Cryptobot-11-30)
✅️ [Antiscam. TON security management](https://telegra.ph/Antiscam-TON-Security-management-12-30)
✅️ [What is DEx and how it works?](https://telegra.ph/DEx--Decentralized-Exchange-01-06)
✅️ [How to create your Jetton](https://telegra.ph/Jetton-Contract-12-12)
✅️ [How to make Validator on TON](https://telegra.ph/How-to-make-Validator-on-TON-12-12)
✅️ [How to become a Nominator](https://telegra.ph/Nominator-contract-in-TON-12-01)
✅️ [How to buy Telegram username](https://telegra.ph/Fragment-marketplace-02-07)

**Comment by inozemtsev-roman** [comment]
![image](https://user-images.githubusercontent.com/24755187/217840888-061217ac-1296-49a3-be48-4495bc29e0be.png)
💎I'm glad to present you the model of head of TON metaspaces.

✅️Soon you could try it
✍️How do you like the concept?
